src/BSON/Value.stub.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+/**
+ * @generate-class-entries static
+ * @generate-function-entries static
+ */
+
+namespace MongoDB\BSON;
+
+final class Value
+{
+    /**
+     * @var int
+     * @cvalue BSON_TYPE_EOD
+     */
+    public const TYPE_EOD = 0x00;
```

Comment:
This is copy-pasta and included for the sake of completeness, but we may want to exclude the `EOD` type as it's irrelevant for users.